portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/openidconnect/OpenIDConnect.testcase

```diff
@@ -455,6 +455,69 @@ definition {
 		}
 	}
 
+	test OpenIDConnectWithGoogleCustomParameters {
+		property dummy.socket.proxy.disabled = "true";
+		property test.name.skip.portal.instance = "OpenIDConnect#EnabledOpenIDConnectOnSystemLevel";
+
+		var customTokenRequestParameters = "resource=https://contacts.google.com";
+		var customAuthorizationRequestParameters = "access_type=offline";
+		var openIdConnectClientId = PropsUtil.get("google.client.id.2");
+		var openIdConnectClientSecret = PropsUtil.get("google.client.secret.2");
+		var userEmailAddress = PropsUtil.get("email.address.2");
+		var userPassword = PropsUtil.get("email.password.2");
+
+		task ("Enable DEBUG level logs for OpenIDConnect") {
```

Comment:
As we discussed we could use the GWT prefixes in our task names like here: https://github.com/liferay/liferay-portal/blob/master/modules%2Fapps%2Fcommerce%2Fcommerce-product-test%2Fsrc%2FtestFunctional%2Ftests%2FCPCommerceOrderTypes.testcase#L294-L312

cc @JoyceWang08211